manifests/t/Tencent/QQMusic/20.47/Tencent.QQMusic.locale.zh-CN.yaml

```diff
@@ -0,0 +1,34 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.locale.1.9.0.schema.json
+
+PackageIdentifier: Tencent.QQMusic
+PackageVersion: '20.47'
+PackageLocale: zh-CN
+Publisher: 腾讯科技（深圳）有限公司
+PublisherUrl: https://y.qq.com/
+PublisherSupportUrl: https://y.qq.com/m/client/helper/pc.html
+PrivacyUrl: https://y.qq.com/m/client/intro/privacy.html
+Author: 腾讯音乐娱乐科技（深圳）有限公司
+PackageName: QQ音乐
+#PackageUrl: https://y.qq.com/download/welcome_pc_v15/index.html
+License: 专有软件
+LicenseUrl: https://y.qq.com/i/user_terms.html
+Copyright: Copyright (C) 2008-2024 Tencent. All Rights Reserved 腾讯公司 版权所有
+CopyrightUrl: https://www.tencent.com/zh-cn/statement.html
+ShortDescription: 腾讯公司推出的免费音乐服务
+Description: QQ 音乐是腾讯公司推出的网络音乐平台，是中国互联网领域领先的正版数字音乐服务的领先平台，同时也是一款免费的音乐播放器，始终走在音乐潮流最前端，向广大用户提供流畅的在线音乐和丰富多彩的音乐社区服务。海量乐库在线试听、卡拉 OK 歌词模式、最流行新歌在线首发、超好用的音乐管理，绿钻用户还可享受高品质音乐试听、正版音乐下载、免费空间背景音乐设置、MV 观看等特权。只要您拥有 QQ 音乐播放器，就拥有属于您自己的流行音乐！
+Tags:
+- 专辑
+- 播放器
+- 歌单
+- 歌手
+- 歌曲
+- 音乐
+- 音频
+ReleaseNotes: |-
+  1. 播放器功能优化，新增浅色样式
+  2. 播放条功能优化，支持歌词显示与调节高度
+  3. 播放器内右键菜单新增歌词居中功能
+  4. 其他体验优化

```

Comment:
```suggestion
  1. 臻品母带与 SQ 音质支持省流模式，开启后可节省更多流量
  2. 其他体验优化
```

Reply:
komac seems to have uses the release notes directly from the last version...